go.mod

```diff
@@ -1,12 +1,12 @@
 module github.com/canonical/ofga
 
-go 1.20
+go 1.22
```

Comment:
Can we at least use `1.21`? `1.22` is still fairly new, and all the projects will have to use this version, if imported.

Reply:
Didn't think of that. Thanks for the suggestions!